cmd/web-handlers.go

```diff
@@ -225,30 +228,26 @@ func (web *webAPIHandlers) ListObjects(r *http.Request, args *ListObjectsArgs, r
 	default:
 		return errAuthentication
 	}
-	marker := ""
-	for {
-		lo, err := objectAPI.ListObjects(args.BucketName, args.Prefix, marker, "/", 1000)
-		if err != nil {
-			return &json2.Error{Message: err.Error()}
-		}
-		marker = lo.NextMarker
-		for _, obj := range lo.Objects {
-			reply.Objects = append(reply.Objects, WebObjectInfo{
-				Key:          obj.Name,
-				LastModified: obj.ModTime,
-				Size:         obj.Size,
-				ContentType:  obj.ContentType,
-			})
-		}
-		for _, prefix := range lo.Prefixes {
-			reply.Objects = append(reply.Objects, WebObjectInfo{
-				Key: prefix,
-			})
-		}
-		if !lo.IsTruncated {
-			break
-		}
+	lo, err := objectAPI.ListObjects(args.BucketName, args.Prefix, args.Marker, "/", 100)
```

Comment:
why only 100, 1000 might be better since for the most part we don't have to do secondary queries.  Perhaps this can be controlled by the browser UI? basically browser can intelligently request on how much it wants?